tests/atomic/main.yml

```diff
@@ -0,0 +1,527 @@
+---
+# vim: set ft=ansible:
+#
+- name: Atomic - Setup
+  hosts: all
+  become: yes
+
+  tags:
+    - setup
+
+  vars_files:
+    - vars.yml
+
+  roles:
+    # This playbook requires Ansible 2.2 and an Atomic Host
+    - role: ansible_version_check
+      avc_major: "2"
+      avc_minor: "2"
+      tags:
+        - ansible_version_check
+
+    # Subscribe if the system is RHEL
+    - role: redhat_subscription
+      when: ansible_distribution == 'RedHat'
+      tags:
+        - redhat_subscription
+
+- name: Atomic Fully Qualified Name
+  hosts: all
+  become: yes
+
+  tags:
+    - fq_name
+
+  vars_files:
+    - vars.yml
+
+  pre_tasks:
+    - name: Set cockpit container name for RHELAH
+      set_fact:
+        cockpit_cname: "registry.access.redhat.com/rhel7/cockpit-ws"
+      when: ansible_distribution == "RedHat"
+
+    - name: Set cockpit container name for Fedora/CentOS
+      set_fact:
+        cockpit_cname: "registry.fedoraproject.org/f25/cockpit"
+      when: "'CentOS' in ansible_distribution or ansible_distribution == 'Fedora'"
+
+  roles:
+    - role: atomic_pull
+      apl_image: "{{ cockpit_cname }}"
+
+    - role: atomic_images_list_verify
+      expected_values:
+        repo: "{{ cockpit_cname }}"
+
+    - role: atomic_containers_list_verify
+      expected_values:
+        image_name: "{{ cockpit_cname }}"
+      expect_missing: true
+
+    - role: atomic_install
+      ai_image: "{{ cockpit_cname }}"
+
+    - role: atomic_run
+      ar_image: "{{ cockpit_cname }}"
+
+    - role: atomic_images_list_verify
```

Comment:
Is it necessary to keep checking that the `cockpit` image is still there after each operation?

Reply:
I can drop this this check.

Reply:
Still looks like there a bunch of extra image checks in the test.  Could you audit the entire test and see if we really need each instance of `atomic_images_list_verify`?